src/Storages/StorageInMemoryMetadata.h

```diff
@@ -52,6 +54,8 @@ struct StorageInMemoryMetadata
     /// structure from different threads. It should be used as MultiVersion
     /// object. See example in IStorage.
 
+    void setComment(const String & comment_);
+
     /// Sets only real columns, possibly overwrites virtual ones.
     void setColumns(ColumnsDescription columns_);
```

Comment:
May be add a comment like `Sets a user-defined comment for a table`

Reply:
Ok.